doc/admin/administration.md

```diff
@@ -87,38 +87,93 @@ boro-11:10001:
         State: FAULTY
 ```
 
-## System Operations
+## System Administration
```

Comment:
That's the title of the overall chapter, so we cannot have it duplicated like this.